__tests__/components/Toast.test.tsx

```diff
@@ -7,20 +7,91 @@ describe('Toast', () => {
     const onDismiss = jest.fn();
 
     test('should render the toast component with the message', () => {
-        render(<Toast message="This is a toast message" isVisible={true} timeToShow={5000} onDismiss={onDismiss} />);
+        render(
+            <Toast
+                message="This is a toast message"
+                isVisible={true}
+                timeToShow={5000}
+                onDismiss={onDismiss}
+                type="success"
+            />
+        );
 
         expect(screen.getByText('This is a toast message')).toBeInTheDocument();
+        expect(screen.getByTestId('toast')).toBeInTheDocument();
     });
 
     test('should not call onDismiss function when the timeToShow is not completed', () => {
-        render(<Toast message="This is a toast message" isVisible={true} timeToShow={5000} onDismiss={onDismiss} />);
+        render(
+            <Toast
+                message="This is a toast message"
+                isVisible={true}
+                timeToShow={5000}
+                onDismiss={onDismiss}
+                type="success"
+            />
+        );
 
         expect(onDismiss).not.toHaveBeenCalled();
     });
 
     test('should call onDismiss function when the timeToShow is completed', async () => {
-        render(<Toast message="This is a toast message" isVisible={true} timeToShow={0} onDismiss={onDismiss} />);
+        render(
+            <Toast
+                message="This is a toast message"
+                isVisible={true}
+                timeToShow={0}
+                onDismiss={onDismiss}
+                type="success"
+            />
+        );
 
         await waitFor(() => expect(onDismiss).toHaveBeenCalled());
     });
+
+    test('should render the error toast component with the message', () => {
+        render(
+            <Toast
+                message="This is a toast message"
+                isVisible={true}
+                timeToShow={5000}
+                onDismiss={onDismiss}
+                type="error"
+            />
+        );
+
+        expect(screen.getByText('This is a toast message')).toBeInTheDocument();
+        expect(screen.getByTestId('toast')).toBeInTheDocument();
```

Comment:
in the case of success as well we are checking these 2 lines only. anyway we can differentiate between success, info and failure toasts ?

Reply:
Done